src/Altinn.Platform/Altinn.Platform.Events/Events/Controllers/SubscriptionController.cs

```diff
@@ -0,0 +1,191 @@
+using System.Threading.Tasks;
+using Altinn.Common.AccessToken.Configuration;
+using Altinn.Common.PEP.Interfaces;
+using Altinn.Platform.Events.Authorization;
+using Altinn.Platform.Events.Configuration;
+using Altinn.Platform.Events.Models;
+using Altinn.Platform.Events.Services.Interfaces;
+using Altinn.Platorm.Events.Extensions;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Altinn.Platform.Events.Controllers
+{
+    /// <summary>
+    /// Controller to handle administration of event subscriptions
+    /// </summary>
+    [Route("events/api/v1/subscription")]
+    [ApiController]
+    public class SubscriptionController : ControllerBase
+    {
+        private readonly IEventsSubscriptionService _eventsSubscriptionService;
+        private readonly IRegisterService _registerService;
+        private readonly ILogger _logger;
+        private readonly string _eventsBaseUri;
+        private readonly AuthorizationHelper _authorizationHelper;
+        private readonly AccessTokenSettings _accessTokenSettings;
+
+        private const string OrganizationPrefix = "/organization/";
+        private const string PersonPrefix = "/person/";
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SubscriptionController"/> class.
+        /// </summary>
+        public SubscriptionController(
+            IEventsSubscriptionService eventsSubscriptionService,
```

Comment:
Would it be enough to use subscriptionService?? This goes for the whole implementation. EventsSubscription seems like "smør på flesk" as we are already in the events component. Do we envision other types of subscriptions being included in this component?